### PR TITLE
feat(scheduler): add a few Kamon metrics

### PR DESCRIPTION
**Pull request by aurrelhebert**

### CODE REVIEW

File: app/models/Order.scala [hunk 1]
```diff
@@ -38,6 +40,7 @@ object Order {
   val dnsOrderFormat = Json.format[DNSOrder]
   val sslOrderFormat = Json.format[SSLOrder]
   implicit val orderWrites = new Writes[Order] {
+    Kamon.gauge("scheduler_order_writes").increment()
```
Comment (urcadox) [inline app/models/Order.scala]: Hm this is just a json serializer; not sure it's relevant to have a metric for this?

File: build.sbt [hunk 2]
```diff
@@ -1,7 +1,7 @@
 name := """poke-api"""
 organization := "com.clever-cloud"
 
-version := "1.0-SNAPSHOT"
+version := "1.1-SNAPSHOT"
```
Comment (urcadox) [inline build.sbt]: I'd rather we do a release of the 1.0 which I can do in another commit once this is merged

**Review by urcadox — CHANGES_REQUESTED** [review]

**Review by StevenLeRoux — APPROVED** [review]

File: app/models/Order.scala (hunk 1 above)
Comment (aurrelhebert) [reply app/models/Order.scala]: Ok thanks :) then it's not needed actually let's remove it

File: build.sbt (hunk 2 above)
Comment (aurrelhebert) [reply build.sbt]: Done :)

**Review by urcadox — APPROVED** [review]

**Pull request merged**